packages/cfstream/cfstream.1.3.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+homepage: "https://github.com/biocaml/cfstream"
+dev-repo: "https://github.com/biocaml/cfstream.git"
+bug-reports: "https://github.com/biocaml/cfstream/issues"
+license: "LGPL + linking exception"
+maintainer: "Ashish Agarwal <agarwal1975@gmail.com>"
+authors: [
+  "Philippe Veber <philippe.veber@gmail.com>"
+  "Ashish Agarwal <agarwal1975@gmail.com>"
+  "Drup <drupyog@zoho.com>"
+]
+
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "jbuilder" {build}
+  "core_kernel" {>= "v0.11.0"}
+  "ounit" {test}
+]
+
+available: [
+  ocaml-version >= "4.06.0"
```

Comment:
Is ther a reason you need this constraint ? I tried using 4.05.0 and it compiled fine.

Reply:
Thanks for noticing this. In fact, the package compiles fine with ocaml 4.04.1. I fixed this constraint.